lib/elixir/test/elixir/kernel/diagnostics_test.exs

```diff
@@ -684,6 +686,90 @@ defmodule Kernel.DiagnosticsTest do
     end
   end
 
+  describe "Code.print_diagnostic" do
```

Comment:
The Elixir suite shouldn't really know about mix, so we should use the diagnostics returned by Code.with_diagnostics (i.e. just a map).

Reply:
[3a4df39](https://github.com/elixir-lang/elixir/pull/12843/commits/3a4df39aecdc5b76dc28bb26ced372d3b6b50f93)

After using gitlab for a while one feature I think github could also have is to automatically link a commit here when it changes a line that was reviewed :smile: